staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -553,8 +553,8 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 		}
 	}
 
-	if feature.DefaultFeatureGate.Enabled(features.RequestManagement) {
-		requestManagementHookName := "generic-apiserver-request-management"
+	requestManagementHookName := "generic-apiserver-request-management"
+	if feature.DefaultFeatureGate.Enabled(features.RequestManagement) && !s.isPostStartHookRegistered(requestManagementHookName) {
```

Comment:
If this change is needed then there may be bigger problems.

Reply:
> 	if c.SharedInformerFactory != nil && !s.isPostStartHookRegistered(genericApiServerHookName) {

i see the other hooks are doing the same.